### DISCUSSION TITLE
PyBricks line following with code

### DISCUSSION THREAD

**Discussion by rajrao** [pybricks/support#1197] [Show and tell]
We were pretty amazed at how well a pybrick based bot followed the line!

[![Watch the video](https://img.youtube.com/vi/LfHmr87trF4/mqdefault.jpg)](https://youtu.be/LfHmr87trF4)
[![Watch the video](https://img.youtube.com/vi/aiKuc22Dqf4/mqdefault.jpg)](https://youtu.be/aiKuc22Dqf4)
[Code](https://github.com/coder-ella/lego/blob/main/pybricks/line_follow_w_drive_base.py)

The code started with some basic line following code we found on this support site and then tweaked.

**Comment by laurensvalk** [comment]
Thanks for sharing this with us. It looks great! :smile:
